src/realm/obj_list.cpp

```diff
@@ -121,3 +121,10 @@ void ObjList::assign(KeyColumn* key_values, const Table* parent)
     m_key_values = key_values;
     m_table = ConstTableRef(parent);
 }
+
+TableVersions ObjList::get_dependency_versions() const
```

Comment:
In ConstTableVIew both functions (taking a parameter/returning value) is named `get_dependencies`. I think we should use the same naming principle in both places - perhaps this one is the better,

This function is not used in Core. Is it used by SDK?

Reply:
It's used by ResultsNotifier in object store. I had the idea that distinguishing the public interface from the actual implementation was a good idea, but maybe not.